### ISSUE TITLE
Missing Channel Form Instructions

### ISSUE THREAD

**Issue by ImpactMind** [elivz/vz_members#6]
Didn't there used to be instructions for using VZ Members as a custom field in channel form, like hidden fields?

**Comment by elivz** [comment]
I'm not clear on exactly what you are trying to do, but if you want to set the selected member in a VZ Members field using a hidden input, just pass the member_id as the value. Otherwise, if you want the user to be able to select the member themselves, you could use the `{field:field_name}` syntax to output a select input.

Let me know if neither of those answered your question, and I will try again!

**Comment by ImpactMind** [comment]
Yeah I figured that one out, but how to add multiple members via a hidden input? Would the parameters be like 

`<input type="hidden" name="vz_members_field[]" value="1|2|3" >`

This is what seems to be missing from the documentation.

**Comment by elivz** [comment]
You would use multiple inputs for that:

```
<input type="hidden" name="vz_members_field[]" value="1">
<input type="hidden" name="vz_members_field[]" value="2">
<input type="hidden" name="vz_members_field[]" value="3">
```
